### PR TITLE
fix #285019: Tours strings for French shows up untranslated

### PR DESCRIPTION
**Pull request by Jojo-Schmitz**

### CODE REVIEW

**Comment by lasconic** [comment]
My bad ...

📦
👨

No paperbag emoji !?!

File: share/locale/tours_fr.ts [hunk 1]
```diff
@@ -0,0 +1,344 @@
...
+        <comment>note-input</comment>
+        <translation>Pour ajouter une note de la durée sélectionnée, appuyez sur sa lettre ou cliquez sur la portée.
+Appuyez sur Ctrl + Haut / Bas (macOS : Cmd + Haut / Bas) pour changer d&apos;octave.</translation>
+    </message>
+    <message>
+        <source>To add an accidental to a note, press the shortcut Up or Down to raise or lower its pitch,
+    or click one of these icons.</source>
+        <comment>note-input</comment>
+        <translation>Pour ajouter une altération accidentelle à une note, appuyez sur les flèches Haut ou Bas pour monter ou descendre la note d&apos;un demi-ton, 
+ou cliquez sur l&apos;un de icônes appropriés.</translation>
```
Comment (GaspardPO) [inline share/locale/tours_fr.ts]: Should not it be "cliquez sur l'un**e** de**s** icône**s** approprié**e**s"

File: share/locale/tours_fr.ts [hunk 2]
```diff
@@ -0,0 +1,344 @@
...
+    <message>
+        <source>To enter notes with different rhythms in the same measure, use multiple voices.
+    Each staff can have up to four voices.
+    The default is voice 1, and it should be used first for each staff.
+    To enter notes into a different voice, click one of these icons.</source>
+        <comment>note-input</comment>
+        <translation>Pour saisir des notes avec des rythmes différents dans la même mesure, utilisez différentes voix.
+Chaque portée peut contenir jusqu&apos;à quatre voix.
+La voix par défaut est la voix 1 et devrait être utilisée pour chaque portée.
+Pour saisir des notes dans une voix différente, cliquez sur l&apos;un des icônes du sélecteur de voix.</translation>
```
Comment (GaspardPO) [inline share/locale/tours_fr.ts]: same as above

File: share/locale/tours_fr.ts [hunk 3]
```diff
@@ -0,0 +1,344 @@
...
+    </message>
+    <message>
+        <source>This is the score view, where you can view and edit your score.
+    Use your mouse wheel or touchpad to scroll (with Shift to scroll horizontally).
+    If you have multiple scores open, you can switch between them using tabs at the top of your score view.</source>
+        <comment>welcome</comment>
+        <translation>Voici la vue de la partition dans laquelle vous pouvez y voir et modifier votre partition.
```
Comment (GaspardPO) [inline share/locale/tours_fr.ts]: probably the **y** is not needed here.

File: share/locale/tours_fr.ts [hunk 4]
```diff
@@ -0,0 +1,344 @@
...
+    It becomes highlighted to indicate that it is selected.
+    You can use the left and right cursor keys to move the selection through the notes and rests of your score.
+    Tip: if you press the Alt key (Option on macOS) along with the cursor keys you can move through other elements too,
+    and also move up and down from staff to staff.</source>
+        <comment>select-tour</comment>
+        <translation>Pour effectuer une sélection unique, cliquez sur l&apos;élément que vous souhaitez sélectionner. 
+Il est mis en surbrillance pour indiquer qu&apos;il est sélectionné.
+ Vous pouvez utiliser les touches de curseur gauche et droite pour déplacer la sélection dans les notes et les silences de votre partition.
+ Conseil : si vous appuyez sur la touche Alt (Option sur macOS) en même temps que les touches de curseur, vous pouvez également vous déplacer parmi d&apos;autres éléments, 
+ainsi que de haut en bas d&apos;un personnel à l&apos;autre.</translation>
```
Comment (GaspardPO) [inline share/locale/tours_fr.ts]: Here, the "staff" should be translated to "une portée", not "un personnel"

File: share/locale/tours_fr.ts (hunk 1 above)
Comment (Jojo-Schmitz) [reply share/locale/tours_fr.ts]: all these need to get fixed/changed on Transifex. This PR is just to have the ts file in the repo and a qm file in the package, so updating translations can work on it

File: share/locale/tours_fr.ts (hunk 4 above)
Comment (Jojo-Schmitz) [reply share/locale/tours_fr.ts]: outch ;-)

File: share/locale/tours_fr.ts (hunk 1 above)
Comment (GaspardPO) [reply share/locale/tours_fr.ts]: oh, ok!
if I want to help improve the translation, it is here ?
https://www.transifex.com/musescore/musescore/

Comment (Jojo-Schmitz) [reply share/locale/tours_fr.ts]: Yes. I've fixed those 4 strings there meanwhile though

**Pull request merged**